docs/source/federated-types/federated-directives.mdx

```diff
@@ -48,6 +65,20 @@ type Product @key(fields: "id") {
 }
 ```
 
+You can apply multiple `@key` directives to a single entity (to specify multiple valid sets of key fields), _if_ your subgraph library supports repeatable directives:
+
+```graphql {1}
+type Product @key(fields: "upc") @key(fields: "sku") {
+  upc: ID!
+  sku: ID!
+  name: String
+}
+```
+
+To check whether your subgraph library supports repeatable directives, see the `@KEY (MULTI)` item in [Federation-compatible subgraph implementations](../supported-subgraphs/).
```

Comment:
actually we just fixed that in the subgraph repo and added a proper test (previously we were not really checking for it) -> new test is named `repeatable @key` (we still got ~half of implementations to update to the latest schema, once updated we'll fix the docs)

Reply:
Ok! Should I just cut this for now and we can add a corresponding sentence when update is complete?

Reply:
lets change it to the expected `repeatable @key` test as otherwise pretty sure we'll forgot about it (`@key multi` is for checking keys that reference multiple fields). I can partially update the results.